provider/pkg/await/await.go

```diff
@@ -215,6 +217,7 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 					urn:               c.URN,
 					initialAPIVersion: c.InitialAPIVersion,
 					clientSet:         c.ClientSet,
+					informerFactory:   dynamicinformer.NewFilteredDynamicSharedInformerFactory(c.ClientSet.GenericClient, 5*time.Second, c.Inputs.GetNamespace(), nil),
```

Comment:
I'm not sure yet if we want to set the refresh period on the informers, and if so, what the period should be. This will need some more testing/tuning.